src/Sarif/RuleUtilities.cs

```diff
@@ -90,13 +90,11 @@ public static Result BuildResult(FailureLevel level, ResultKind kind, IAnalysisC
             return result;
         }
 
-        public static Dictionary<string, string> BuildDictionary(
+        public static Dictionary<string, MultiformatMessageString> BuildDictionary(
             ResourceManager resourceManager,
             IEnumerable<string> resourceNames,
-            string ruleId,
-            string prefix = null)
```

Comment:
>prefix [](start = 19, length = 6)

the 'prefix' was used to provide default rich text messages. no one has consumed this functionality, which needs to be reauthored.